includes/admin/class-wc-rest-payments-disputes-controller.php

```diff
@@ -147,11 +147,13 @@ public function close_dispute( $request ) {
 	 * @param WP_REST_Request $request Full data about the request.
 	 */
 	public function get_disputes_export( $request ) {
-		$user_email = $request->get_param( 'user_email' );
-		$locale     = $request->get_param( 'locale' );
-		$filters    = $this->get_disputes_filters( $request );
+		$user_email   = $request->get_param( 'user_email' );
+		$wpcom_locale = WC_Payments_Utils::convert_to_server_locale(
+			$request->get_param( 'locale' )
+		);
+		$filters      = $this->get_disputes_filters( $request );
```

Comment:
Nice and simple to do this inline. Long term I wonder if we might want middleware-style generic converters for various standard args.

That is probably overengineering, but at the same time, I worry that our request wrapper handling in client is too manual, since it's almost a 1:1 mapping to the service API, tweaking a few things here and there. One way to document that "magic" is to implement as middleware pattern. For example, WooPayments JavaScript front end calls store PHP/REST API, store PHP generically adds authentication for WooPayments Service (WPCOM) API.